sdk/compute/azure-mgmt-avs/sdk_packaging.toml

```diff
@@ -0,0 +1,5 @@
+[packaging]
+package_name = "azure-mgmt-avs"
+package_pprint_name = "VMWare Solution Management"
```

Comment:
I'm not sure where this shows up, but AVS is Azure VMware Solution. Definitely `VMWare` needs to be changed to `VMware`.

Reply:
Thanks. It will show up at [readme.md](https://github.com/Azure/azure-sdk-for-python/pull/12003/files#diff-cff332b6b062d90ffa2e1dbfdca5e380R3) and [setup.py](https://github.com/Azure/azure-sdk-for-python/pull/12003/files#diff-3f63e183804861601a44c027a87aaea4R16).